apps/backend/src/libs/modules/map-box/libs/enums/map-box-paths.enum.ts

```diff
@@ -0,0 +1,5 @@
+const MapBoxAPIPaths = {
```

Comment:
The enum name should be in singular form, the same for MapBoxAPIProfiles -> MapboxAPIProfile
```suggestion
const MapboxAPIPath = {
```